src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -4,14 +4,15 @@
 import edu.kis.legacy.drawer.shape.ILine;
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
 
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dDriverAdapter  implements Job2dDriver {
 	private int startX = 0, startY = 0;
-
-	public MyAdapter() {
+	private DrawPanelController drawerController  = DrawerFeature.getDrawerController();
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.